tests/PHPUnit/CustomAffixes/FolderWithMultiple@@@@/ExampleTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+class MyFailingCustomClassTest extends PHPUnit\Framework\TestCase
+{
+    public function assertTrueIsTrue()
+    {
+        $this->assertTrue(true);
+    }
+}
+
+uses(MyFailingCustomClassTest::class);
+
+test('custom traits can be used', function () {
```

Comment:
This TestCase is no longer needed and will actually fail completely after the change to `>>>`

Reply:
I just updated the tests to include `>>>` in the folder name, as it seems to work on macOS. I think that keeping the test can help people having the same issue when they have some strange folder names as I had.

Reply:
Out of interest, what would happen if you attempted to pull this repo on a Windows machine?